src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2.java

```diff
@@ -395,7 +396,10 @@ private VariantAnnotatorEngine initializeVCFOutput() {
         // if printTCGAsampleHeader, we already checked for exactly 1 tumor and 1 normal in printTCGAsampleHeader assignment in initialize()
         final List<String> outputSampleNames = printTCGAsampleHeader ? Arrays.asList("TUMOR", "NORMAL") : samplesList.asListOfSamples();
 
-        vcfWriter.writeHeader(new VCFHeader(headerInfo, outputSampleNames));
+        //TODO: figure out what was outputting a null line (there was exactly 1) and remove this hack!!!
```

Comment:
I would guess (the absence of) `clusteredReadPosition` annotator is causing the problem here